Blog: Time management (#83)
rathesDot/rathes.de: resources/contents/blog/en/managing-time/managing-time.md

---
id: 201904081
title: "Managing time"
date: 2019-04-08T12:13:22+02:00
author: Rathes Sachchithananthan
template: post
description: Time Management is not logical but emotional, you don't manage time, you manage yourself.
image: /images/blog/time.jpg
categories:
  - Life
tags:
  - time management
  - productivity
locale: en_US
---

> Many people believe that time management is an effective skill they need to have to be powerful and effective, to build the life they want. Therefore they spend a lot of time investing in tools, courses and workshops to manage their time better. And yet they fail to be more effective and productive.

Author: ```suggestion
Many people believe that time management is an effective skill they need to have to be powerful and effective, to build the life they want. Therefore they spend a lot of time investing in tools, courses, and workshops to manage their time better. And yet they fail to be more effective and productive.
```

It isn't because they are wrong about the importance of time management but it is how they see the topic and how they try to find solutions.

## Time Management is not driven by logic but by emotion

> The truth is that there is not such a thing as time management. Not matter how you see it, time goes on, second for second. You can't manage the time. You will always have 24 hours, 1440 minutes, 86 400 seconds every single day. You can't have more time one day and less on the other, you can move some hours around, it's not possible.

Author: ```suggestion
The truth is that there is not such a thing as time management. No matter how you see it, time goes on, second for second. You can't manage the time. You will always have 24 hours, 1440 minutes, 86 400 seconds every single day. You can't have more time one day and less on the other, you can move some hours around, it's not possible.
```

> Logically it is impossible to manage time. And still you can manage time. But from the emotional perspective. You can't control the time but you can control how you spend your time. You can decide when you want to do things and when you want to do other things or even none at all. So time management is more self management.

Author: ```suggestion
Logically, it is impossible to manage time. And still, you can manage time. But from the emotional perspective. You can't control the time but you can control how you spend your time. You can decide when you want to do things and when you want to do other things or even none at all. So time management is more self-management.
```

## You will find time if you have to

There is no excuse about not having time. You can control all 86 400 seconds of your day. Let me give an example.

> Last year, during winter, our engineering team went out for a little field trip. It was freezing cold, still it was a wonderful trip. But when I returned I got the shock of my life.

Author: ```suggestion
Last year, during winter, our engineering team went out for a little field trip. It was freezing cold, still, it was a wonderful trip. But when I returned I got the shock of my life.
```

When I entered my apartment it was freezing cold, my boiler wasn't working. And when I went into the kitchen where it is located I saw a huge mess. My boiler has been leaking water the entire weekend and the kitchen was flooded.

> Without any further delay, I dropped my stuff and started cleaning the kitchen. I called my landlord to inform her, called the guy who was in charge of the boiler, arranged an appointment for the next morning before work and ensured that he was able to enter my apartment while I was at work. It took me like 4 to 5 hours on a Sunday but in the end I got things fixed and it wasn't too bad.

Author: ```suggestion
Without any further delay, I dropped all my plans and started cleaning the kitchen. I called my landlord to inform her, called the guy who was in charge of the boiler, arranged an appointment for the next morning before work and ensured that he was able to enter my apartment while I was at work. It took me like 4 to 5 hours on a Sunday but in the end, I got things fixed and it wasn't too bad.
```

> If someone had called me that Sunday to ask if I had four or five hours, do you think I would have said that I had time? Definitely not. But still I had the time to do things that were totally unplanned.

Author: ```suggestion
If someone had called me that Sunday to ask if I had four or five hours, do you think I would have said that I had time? Definitely not. But still, I had the time to do things that were totally unplanned.
```

## Time management is about priorities

> I did find that time because I had to find it. If I had waited longer, it would have ended worse and have costed me a lot of money. From that point when I saw that mess it instantly became my highest priority and thus I had to make time to do things that had to be done.

Author: ```suggestion
I did find that time because I had to find it. If I had waited longer, it would have ended worse and have cost me a lot of money. From that point when I saw that mess, it instantly became my highest priority and thus I had to make time to do things that had to be done. 
```

And like I found time to fix the boiler in my apartment, you can also make time to do the things that matter. Put those things on top of your schedule and do them first.

### How to find the priorities?

So to manage your time properly, you need to find the right priorities. How would you do that?

> First of all, you need to find out your overall goals. What do you want to achieve? What are your desires? Think not only about professional goals, but also about personal.

Author: ```suggestion
First of all, you need to find out your overall goals. What do you want to achieve? What are your desires? Think not only about professional goals but also about personal. 
```

> Every quarter of a year, I have a small session that I call a _life review_. I just spend some time on my couch and I go through my small notebook in which I note down all my thoughts and ideas. I check if I follow what I wanted to do, look for reason why things went well or why the didn't and come up with some action points for the next months. I also adjust my goals if needed.

Author: ```suggestion
Every quarter of a year, I have a small session that I call a _life review_. I just spend some time on my couch and I go through my small notebook in which I note down all my thoughts and ideas. I check if I follow what I wanted to do, look for the reason why things went well or why the didn't and come up with some action points for the next months. I also adjust my goals if needed. 
```

> I have three different types of goals: Professional goals, personal life goals and relationship goals. And I try to keep each of those types on track with tasks on a daily basis.

Author: ```suggestion
I have three different types of goals: Professional goals, personal life goals, and relationship goals. And I try to keep each of those types on track with tasks on a daily basis. 
```

### What is priority

> If you know what your goals are and what you have to do in order to reach them it is time to execute. Execute the hightest priority tasks first and the go down the list.

Author: ```suggestion
If you know what your goals are and what you have to do in order to reach them it is time to execute. Execute the highest priority tasks first and then go down the list. 
```

But how exactly do you find the correct priority? Many of you have heard of the Eisenhower principle. That principle focuses on effectiveness and efficiency. There are basically two questions that you ask:

1. **Importance**: How much does it matter?
2. **Urgency**: How soon does it matter?

>    Every task that has been identified as important and urgent are the highest priority, followed by the ones that are important but not that urgent. The you do urgent but not that important tasks and in the end you can check the unimportant and not that urgent tasks.

Author: ```suggestion
   Every task that has been identified as important and urgent are the highest priority, followed by the ones that are important but not that urgent. Then, you do urgent but not that important tasks and in the end, you can check the unimportant and not that urgent tasks.
```

### The third dimension: Significance

While the Eisenhower principle works fine there is something missing that makes the difference between people the ones that manage time pretty well and those who seem to have more time than others.
It is the third dimension that should define your priority:

3. Significance: How long does it matter?

>    This additional dimension will make sure that you work on things that will make your more effective and efficient on the very long term. Especially in the modern world this third parameter becomes more and more important.

Author: ```suggestion
   This additional dimension will make sure that you work on things that will make your more effective and efficient on the very long term. Especially in the modern world, this third parameter becomes more and more important. 
```

1. Can I elimiate this?

   Is this even important enough to it or did it became obsolete over time? Don't do things that do have any impact.

2. Can I automate it?

   Is there a way that you can automate the task? So that you don't have to do it in the future anymore? Especially nowadays there are a lot of things that can be automated, use the technologies.

3. Can I delegate it?

   You don't have to do every single thing. It might be hard to get into it in the beginning but you do not always have to do things on your own. The best example for freelancers is bookkeeping: Yes you can do it yourself, it is not that complex once you understood it. But is the money you save worth it compared to the time you spend on it? Most of the time it is better to delegate the task to someone who is specialized in that topic.

   Another example is when software companies start building internal tools instead of just using an existing service. In almost all cases it is more efficient to go for an existing solution and invest the saved time and money in the product that you sell.

4. Does it have to happen now or can it be done later?

>    Decide whether you have to do the task immediately or you can postpone it. If it doesn't have to happen now, then put the task back on your list and check the next one. Maybe in a week this task does not matter anymore and you can just eliminate it.

Author: ```suggestion
   Decide whether you have to do the task immediately or you can postpone it. If it doesn't have to happen now, then put the task back on your list and check the next one. Maybe in a week, this task does not matter anymore and you can just eliminate it. 
```

## Tl;dr

> Time Management is not about managing time but managing yourself. You have decide what does matter to you and do those things first. And if you set your priorities of your tasks make sure that you think about the long term impact of them.

Author: ```suggestion
Time Management is not about managing time but managing yourself. You have to decide what does matter to you and do those things first. And if you set the priorities of your tasks, make sure that you think about the long term impact of them.
```
